GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift

```diff
@@ -123,6 +71,72 @@ extension SecureConversations {
             return controller
         }
 
+        private func bindDelegate(
+            to event: WelcomeViewModel.DelegateEvent,
+            controller: WelcomeViewController?
+        ) {
+            let style = viewFactory.theme.secureConversationsWelcomeStyle
+            switch event {
+            case .backTapped:
+                delegate?(.backTapped)
+            case .closeTapped:
+                delegate?(.closeTapped)
+            // Bind changes in view model to view controller.
+            case let .renderProps(props):
+                controller?.props = props
+            case .confirmationScreenRequested:
+                presentSecureConversationsConfirmationViewController()
+            case let .mediaPickerRequested(originView, callback):
+                controller?.presentPopover(
+                    with: style.pickMediaStyle,
+                    from: originView,
+                    // Designs use 'up' arrow, but currently
+                    // it seems like there is a bug in
+                    // AttachmentSourceListView, that makes
+                    // it render incorrectly with 'up' arrow.
+                    // That is why using 'down' arrow for now.
+                    arrowDirections: .down,
+                    itemSelected: { [weak controller] kind in
+                        controller?.dismiss(animated: true)
+                        callback(kind)
+                    }
+                )
+            case let .pickMedia(callback):
+                presentMediaPickerController(
+                    with: callback,
+                    mediaSource: .library,
+                    mediaTypes: [.image, .movie]
+                )
+            case let .takeMedia(callback):
+                presentMediaPickerController(
+                    with: callback,
+                    mediaSource: .camera,
+                    mediaTypes: [.image, .movie]
+                )
+            case let .pickFile(callback):
+                presentFilePickerController(with: callback)
+            case let .showFile(file):
+                presentQuickLookController(with: file)
+            case let .showAlert(conf, accessibilityIdentifier, dismissed):
+                controller?.presentAlert(
+                    with: conf,
+                    accessibilityIdentifier: accessibilityIdentifier
+                ) { dismissed?() }
```

Comment:
Could we do similar fix here also?
```suggestion
                controller?.presentAlert(
                    with: conf,
                    accessibilityIdentifier: accessibilityIdentifier,
                    dismissed: dismissed
                )
```